crates/biome_rowan/src/syntax_node_text.rs

```diff
@@ -88,6 +88,29 @@ impl SyntaxNodeText {
         }
     }
 
+    pub fn starts_with(&self, mut prefix: &str) -> bool {
```

Comment:
I didn't end up using this method in the PR, but I wrote it during an earlier approach I tried for this PR. Technically it's dead code, but I figured it might come in handy still.